src/master/develop/connection.md

```diff
@@ -7,13 +7,15 @@ You can modify defaults on the immudb server in `immudb.toml` in the config fold
 ::: tab Go
 
 ```go
-c, err := client.NewImmuClient(client.DefaultOptions())
-if err != nil {
-    log.Fatal(err)
-}
-ctx := context.Background()
-// login with default username and password
-_ , err = c.Login(ctx, []byte(`immudb`), []byte(`immudb`))
+import (
+  "context"
+  immudb "github.com/codenotary/immudb/pkg/client"
+  "log"
+)
+```
```

Comment:
Minor: There are some markdown linting errors in this file, if using vscode, this one is a pretty nice tool to find all issues: https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint

Reply:
I added markdownlint to precommit hooks list, ty